### DISCUSSION TITLE
getValuesIn方法不生效，求解答

### DISCUSSION THREAD

**Discussion by huaxiangsirenbutong** [alibaba/formily#2788] [Q&A 2.x] [answered]
getValuesIn方法不生效，求助！！
  输出如下
  console.log(form.getValuesIn('MobileSearchBar.MobileSearchForm.MobileSearchObject.DOCTYPE'))
![image](https://user-images.githubusercontent.com/37055241/150754485-58e6515a-cd0a-4358-8d18-53c9b447e82b.png)
 JSONSCHEMA如下
![image](https://user-images.githubusercontent.com/37055241/150754749-a07d89ec-3c6a-4b7d-a926-2cbdce14d41a.png)
@cncolder @s0ngyee @zb188

**Comment by huaxiangsirenbutong** [comment]
getFieldState方法可以正常使用，getValuesIn不行换了别的表单值测试也是不行 ，求解答

**Comment by janryWang** [comment, marked as answer]
因为存在void节点吧，void是不占数据路径的

**Reply by huaxiangsirenbutong** [reply]
谢谢，试了下是这样
